masterlist.yaml

```
@@ -8170,6 +8170,33 @@ plugins:
     url: [ 'https://www.nexusmods.com/oblivion/mods/26570/' ]
     group: *mapGroup
 
+  - name: 'Map Marker Overhaul( - SI additions)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/26389/'
+        name: 'Map Marker Overhaul'
+  - name: 'Map Marker Overhaul.esp'
+    inc:
+      - 'Landmarks, w Wells.esp'
+      - 'Oblivion Gate Map Marker Remover.esp'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Thieves Arsenal'
+          - '[Thieves Arsenal](https://www.nexusmods.com/oblivion/mods/9655/)'
+        condition: 'active("Thieves Arsenal.esp") and not file("Thieves_Arsenal_Patch_033010.esp")'
+      - <<: *patchProvided
```

The patch is in the mod's archive, so `*patchIncluded` is more appropriate.

```suggestion
      - <<: *patchIncluded
```